src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java

```diff
@@ -13,22 +17,31 @@
 @Controller
 public class MessageWebSocketAccess {
 
+    @Autowired
+    private MessageRepository messageRepository;
     private static final Gson gson = new Gson();
     private MessageInteractor messageInteractor = new MessageInteractor();
 
     @MessageMapping("/message-customer/{sessionId}")
     @SendTo("/topic/message-employee/{sessionId}")
     public String sendMessageCustomer(@DestinationVariable String sessionId, String message) throws Exception {
+        sendResponseToDatabase(message, sessionId, Sender.CUSTOMER);
         HashMap<String, String> response = messageInteractor.generateResponse(message);
         return gson.toJson(response);
     }
 
     @MessageMapping("/message-employee/{sessionId}")
     @SendTo("/topic/message-customer/{sessionId}")
     public String sendMessageEmployee(@DestinationVariable String sessionId, String message) throws Exception {
+        sendResponseToDatabase(message, sessionId, Sender.EMPLOYEE);
         HashMap<String, String> response = messageInteractor.generateResponse(message);
         return gson.toJson(response);
     }
 
 
+    private void sendResponseToDatabase(String content, String sessionId, Sender sender){
```

Comment:
Can you place this method in another file?

Reply:
done moved to MessageDataAccess class